### ISSUE TITLE
Doctrine 3

### ISSUE THREAD

**Issue by cheack** [jsor/doctrine-postgis#75]
Is there a chance we'll see support for Doctrine ORM 3 and DBAL 4 anytime soon?
Thanks.

**Comment by JoDerGraf** [comment]
I've completed the update. The integration of Doctrine ORM 3 and DBAL 4 is now complete. [https://github.com/jsor/doctrine-postgis/pull/78](https://github.com/jsor/doctrine-postgis/pull/78)

**Comment by maxhelias** [comment]
Yes it's merge. I finished this PR : https://github.com/jsor/doctrine-postgis/pull/67 and I made a release
